decidim-participatory_processes/app/queries/decidim/participatory_processes/stats_followers_count.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    # This class counts all Followers of a participatory processes
+    class StatsFollowersCount < Rectify::Query
+      def self.for(participatory_space)
+        return 0 unless participatory_space.is_a? Decidim::ParticipatoryProcess
+
+        new(participatory_space).query
+      end
+
+      def initialize(participatory_space)
+        @participatory_space = participatory_space
+      end
+
+      def query
+        space_query + components_query
+      end
+
+      private
+
+      def components_query
+        Decidim.component_manifests.sum do |component|
+          component.stats
+                   .filter(tag: :followers)
+                   .with_context(@participatory_space.components.published)
+                   .map { |_name, value| value }
+                   .sum
+        end
+      end
+
+      def space_query
+        Decidim.participatory_space_manifests.sum do |space|
+          space.stats
+               .filter(tag: :followers)
+               .with_context(@participatory_process)
```

Comment:
Should this be `@participatory_space` instead?

If this is the case, I usually add an `attr_reader` for each instance variable in a class, and then use the method instead of calling the instance variable directly. Calling an undefined method raises an error, but calling an undefined instance variable just returns `nil`. Thus, adding the `attr_reader` and calling the method it defines helps avoid typos 😄

Reply:
Thanks, I'll fix it 🔧